lldb/source/Commands/Options.td

```diff
@@ -387,8 +387,13 @@ let Command = "expression" in {
     Arg<"Boolean">,
     Desc<"Controls whether the expression can fall back to being JITted if it's "
     "not supported by the interpreter (defaults to true).">;
+  def persistent_result : Option<"persistent-result", "\\x01">, Groups<[1,2]>,
+    Arg<"Boolean">,
+    Desc<"Persist expression result in a variable for subsequent use. "
+    "Expression results will be labeled with $-prefixed variables, e.g. $0, "
+    "$1, etc. Defaults to true.">;
   // BEGIN SWIFT
-  def bind_generic_types : Option<"bind-generic-types", "\\x01">,
+  def bind_generic_types : Option<"bind-generic-types", "\\x02">,
```

Comment:
Could we use a much higher number for this, so it's less likely to break in the future?

Reply:
Good idea.

Reply:
Switched to `\x31`, the highest control code (Unit Separator) and last char value prior to the first printable character (space character).

Reply:
cc @augusto2112

Reply:
Yes I think that's fine!